package.json

```diff
@@ -72,6 +72,7 @@
     "dayjs": "^1.11.10",
     "ember-async-data": "^1.0.3",
     "ember-auto-import": "^2.6.3",
+    "ember-changeset": "^4.1.2",
```

Comment:
still need this import? does not seems used explictely

Reply:
Is that a question directed at me particularly? In any case, I'm not really sure if this is used for something else. Maybe @sergiofenoll can give his input?

Reply:
I don't see any imports of this package, so perhaps it can go, unsure.